org.jacoco.core.test/src/org/jacoco/core/instr/InstrumenterTest.java

```diff
@@ -97,17 +97,18 @@ public void setup() throws Exception {
 	}
 
 	@Test
-	public void should_instrument_java10_class() throws Exception {
-		final byte[] originalBytes = createClass(BytecodeVersion.V10);
+	public void should_instrument_class_greater_than_max_version()
```

Comment:
While "downgrade" trick can be removed, this test must be preserved under another name to prevent regressions, because it verifies that `Instrumenter` computes `classId` using original class bytes.